freeipa.tf

```diff
@@ -24,22 +24,32 @@ module "security_groups" {
     aws = aws.sharedservicesprovisionaccount
   }
 
+  load_balancer_ips   = [for i in data.aws_network_interface.nlb : i.private_ip]
   trusted_cidr_blocks = var.trusted_cidr_blocks
   vpc_id              = data.terraform_remote_state.networking.outputs.vpc.id
 }
 
 # Create the IPA servers
-module "ipa0" {
+module "ipa" {
   source = "github.com/cisagov/freeipa-server-tf-module"
+  # AWS reserves the first four and the last IP address in each
+  # subnet.
+  #
+  # cisagov/freeipa-server-tf-module now requires us to assign IPs in
+  # order to break the dependency of DNS record resources on the
+  # corresponding EC2 instance resources; otherwise, it is not
+  # possible to recreate the IPA servers one by one as is required
+  # when a new FreeIPA AMI is made available.
```

Comment:
Did you mean to have this comment both above (in locals) and repeated here?

Reply:
Please see commit 66de4789f6817e69e68281d0f3868cac51dfa79e.